src/components/Navigation.vue

```diff
@@ -0,0 +1,167 @@
+<template>
```

Comment:
Components should be multiworded e.g. Navigation (Bad) --> TheNavigation (Good) https://vuejs.org/v2/style-guide/#Multi-word-component-names-essential
Components that are used once in the application like Navigation should be prefixed with "The" e.g. "TheNavigation" https://vuejs.org/v2/style-guide/#Single-instance-component-names-strongly-recommended